engine/schema/src/main/resources/META-INF/db/schema-41810to41900.sql

```diff
@@ -314,3 +314,8 @@ CREATE TABLE `cloud_usage`.`bucket_statistics` (
   `size` bigint unsigned COMMENT 'total size of bucket objects',
    PRIMARY KEY(`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- Add remover account ID to quarantined IPs table.
+ALTER TABLE `cloud`.`quarantined_ips`
+    ADD COLUMN `remover_account_id` bigint(20) unsigned DEFAULT NULL COMMENT 'ID of the account that removed the IP from quarantine, foreign key to `account` table',
+    ADD CONSTRAINT `fk_quarantined_ips__remover_account_id` FOREIGN KEY(`remover_account_id`) REFERENCES `cloud`.`account`(`id`);
```

Comment:
should these be made idempotent? so if somebody decides to backport the entire IP quarantine to an older branch they can still upgrade? I understand this might be questionable but the coding is not a big deal and is good in principle.